### PR TITLE
feat: Added Card components

### PR DESCRIPTION
**Pull request by kjais1720**
- Added Card components
- Added some utility classes
- Refactored some code
- preview-link: https://deploy-preview-13--peaceful-golick-077be9.netlify.app/docs/cards/car
image

### CODE REVIEW

File: stylesheets/components/cards.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
+/* .cards-container{
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(18.75em, 1fr));
+    align-items: center;
+    justify-items: center;
+    gap: 1.25rem;
+} */
```
Comment (rdshinde) [inline stylesheets/components/cards.css:7]: If this is not needed you can remove it instead of commenting out.

**Review by rdshinde — COMMENTED** [review]
Please try to add some comments separating the code of different parts of card.
It will be very helpful for readers and reviewers while going through the code.

File: stylesheets/components/cards.css (hunk 1 above)
Comment (rdshinde) [reply stylesheets/components/cards.css:7]: Otherwise all looks good. 👍

File: stylesheets/components/cards.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+.tr-card-badge{
+    position: absolute;
+    font-size: .9rem;
+    top: 1.0rem;
+    left: 0;
+    padding: 5px var(--space-md);
+    background-color: var(--tr-accent-color) ;
+    color: var(--tr-text);
+}
```
Comment (vivekmehto) [inline stylesheets/components/cards.css:62]: you can use badge component here.

**Review by vivekmehto — COMMENTED** [review]
Looks perfect just one suggestion you can use the badge component for a badge in the card.
And that preview link is broke I think.

File: stylesheets/components/cards.css (hunk 2 above)
Comment (kjais1720) [reply stylesheets/components/cards.css:62]: I haven't created the badge components yet, I will extract this in the badge.css once I create it

File: stylesheets/components/cards.css (hunk 1 above)
Comment (kjais1720) [reply stylesheets/components/cards.css:7]: Thank you for pointing it out, I've removed it in the latest commit

**Review by vivekmehto — APPROVED** [review]

**Review by rdshinde — APPROVED** [review]
Looks good now.

**Pull request merged**